backend/coins/btc/handlers/handlers.go

```diff
@@ -364,6 +365,51 @@ func (handlers *Handlers) getAccountBalance(*http.Request) (interface{}, error)
 	}, nil
 }
 
+type slip24Request struct {
+	RecipientName string `json:"recipientName"`
+	Nonce         string `json:"nonce"`
+	Memos         []struct {
+		Type string `json:"type"`
+		Text string `json:"text"`
+	} `json:"memos"`
+	Outputs []struct {
+		Amount  uint64 `json:"amount"`
+		Address string `json:"address"`
+	} `json:"outputs"`
+	Signature string `json:"signature"`
+}
+
+func (slip24 slip24Request) toPaymentRequest() (*accounts.PaymentRequest, error) {
+	if len(slip24.Outputs) != 1 {
+		return nil, errp.New("Missing or multiple payment request output unsupported")
+	}
+
+	if len(slip24.Nonce) > 0 {
+		return nil, errp.New("Nonce value unsupported")
+	}
+
+	sigBytes, err := hex.DecodeString(slip24.Signature)
```

Comment:
Per README example, request-address's signature is a base64 encoded DER-encoded signature (~70 bytes).

(The BitBox02 takes a compact 64 byte encoding)